scripts/kind-e2e/e2e.sh

```diff
@@ -51,7 +51,7 @@ function setup_broker() {
         echo Submariner CRDs already exist, skipping broker creation...
     else
         echo Installing broker on cluster1.
-        helm install submariner-latest/submariner-k8s-broker --name ${SUBMARINER_BROKER_NS} --namespace ${SUBMARINER_BROKER_NS}
+        helm install submariner-latest/submariner-k8s-broker --name ${SUBMARINER_BROKER_NS} --namespace ${SUBMARINER_BROKER_NS} > /dev/null 2>&1
```

Comment:
Same here (and in other places further down), please keep the output. (Even if we do end up wanting to get rid of it, it should be a separate patch.)

Reply:
> Same here (and in other places further down), please keep the output. (Even if we do end up wanting to get rid of it, it should be a separate patch.)

Yes i agree

Reply:
Done with latest commit.